types/uuid/interfaces.d.ts

```diff
@@ -48,3 +48,12 @@ export interface v5Static {
     URL: string;
 }
 export type v5 = v5Buffer & v5String & v5Static;
+
+// https://github.com/uuidjs/uuid/blob/master/src/parse.js
+export type parse = (uuid: string) => Uint8Array;
```

Comment:
It's an implementation detail that we are currently returning a Uint8Array here, this should not be relied upon. Instead this should be an `OutputBuffer`.